perftests/network-ssl/test_network_ssl.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+./test_network_ssl www.tarsnap.com
```

Comment:
We should have some tests which fail here.  In fact making sure that things like `wrong.host.badssl.com` fail is far more important than making sure that the right things succeed, as far as testing is concerned!